LICENCE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [2023] [Stratego]
```

Comment:
The name is Statego, unfortunately we do not have the copyright to the game Stratego